hudi-cli/src/main/java/org/apache/hudi/cli/commands/SavepointsCommand.java

```diff
@@ -63,36 +63,36 @@ public String showSavepoints() throws IOException {
   }
 
   @CliCommand(value = "savepoint create", help = "Savepoint a commit")
-  public String savepoint(@CliOption(key = {"commit"}, help = "Commit to savepoint") final String commitTime,
+  public String savepoint(@CliOption(key = {"commit"}, help = "Commit to savepoint") final String instantTime,
```

Comment:
this is specifically doing a savepoint for a commit.. so `commitTime` is actually apt

Reply:
Thanks, I will fix it